src/Impostor.Api/Net/Messages/S2C/Message16GetGameListS2C.cs

```diff
@@ -25,10 +25,10 @@ public static void Serialize(IMessageWriter writer, int skeldGameCount, int mira
                 writer.Write(game.PublicIp.Address);
                 writer.Write((ushort)game.PublicIp.Port);
                 writer.Write(game.Code);
-                writer.Write(game.Host.Client.Name);
+                writer.Write(game.DisplayName ?? game.Host.Client.Name);
```

Comment:
Add a Default value to DisplayName because it throughs an error when a client requests the game list while you didn`t set the display name while creating the game